src/renderers/shaders/ShaderChunk/clipping_planes_fragment.glsl

```diff
@@ -0,0 +1,11 @@
+#if NUM_USER_CLIPPING_PLANES > 0
+
+	for ( int i = 0; i < NUM_USER_CLIPPING_PLANES; ++ i ) {
+
+		vec4 plane = userClippingPlanes[ i ];
+		if ( dot( vViewPosition, plane.xyz ) > plane.w ) discard;
```

Comment:
Is there clipping support in ES 3? I know there is in OpenGL 4... If so, we can just keep the infrastructure, and set an appropriate `gl_ClipDistance` at vertex level instead.

For a math library, it should probably be `signedPlaneDistance`, so the factorization has a factor > 1.